atom/browser/browser_mac.mm

```diff
@@ -36,8 +36,15 @@
   [[AtomApplication sharedApplication] hide:nil];
 }
 
-void Browser::Show() {
-  [[AtomApplication sharedApplication] unhide:nil];
+void Browser::Show(mate::Arguments* args) {
+  bool should_activate;
+  args->GetNext(&should_activate);
```

Comment:
this doesn't match the docs, which say the argument should be an object

Reply:
oops yeah sorry this should say wip :) still working out some things